deployments/ansible/molecule/default/Dockerfile.j2

```diff
@@ -0,0 +1,31 @@
+{% if item.image not in ["opensuse12", "opensuse15"] %}
+FROM geerlingguy/docker-{{ item.image }}-ansible:latest
+{% else %}
+{% if item.image == "opensuse12" %}
+FROM opensuse/leap:42
+RUN zypper -n install -l ansible dbus-1 rpm-python sudo systemd-sysvinit
+{% else %}
+FROM opensuse/leap:15
+RUN zypper -n install -l ansible dbus-1 python3-rpm sudo systemd-sysvinit
+{% endif %}
+
+ENV container docker
+
+RUN (cd /usr/lib/systemd/system/sysinit.target.wants/; for i in *; do [ $i = \
+    "systemd-tmpfiles-setup.service" ] || rm -f $i; done); \
+    rm -f /usr/lib/systemd/system/multi-user.target.wants/*;\
```

Comment:
Is this `/usr/lib/systemd` on all platforms or just suse?

Reply:
oh, missed the parent`if`...

Reply:
Yeah, the idea was to use custom images for opensuse, and just use the pre-built `geerlingguy` ansible images for all other distros.